src/index.tsx

```diff
@@ -66,6 +72,7 @@ export default function Wordcloud({
   options,
   size: initialSize,
   words,
+  renderDebounce = 100,
```

Comment:
Let's integrate this in `options` prop.  At some point I had the choice to use flat props or organize configuration related props in an `options` object.  I chose the latter, so I think that's the right place for this to go in.  Don't worry too much about updating the documentation.  If you update the `propTypes` comment string, that should show up at least in the `Props` documentation page.